### ISSUE TITLE
Check for git repo when changing root

### ISSUE THREAD

**Issue by cemsbr** [Xuyuanp/nerdtree-git-plugin#82]
If I open vim in a regular directory with several git repos inside and change the NERDTree root to a repo dir, git status is not show even after a refresh.

**Comment by al-x** [comment]
When I had this same problem, I was able to fix it by adding the following line to my ```vimrc```:

```
let NERDTreeChDirMode = 2
```
Then, when changing NERDTree root (ie, typing ```C``` (```NERDTreeMapChangeRoot```) with a selection), you must press ```R``` (```NERDTreeMapRefreshRoot```) to refresh the visual git markings for that git repo.

I figured this out by reading through the ```:help 'NERDTreeChDirMode'```:
```
Values: 0, 1 or 2.
Default: 0.

Use this option to tell the script when (if at all) to change the current
working directory (CWD) for vim.

If it is set to 0 then the CWD is never changed by the NERD tree.

If set to 1 then the CWD is changed when the NERD tree is first loaded to the
directory it is initialized in. For example, if you start the NERD tree with
    :NERDTree /home/marty/foobar

then the CWD will be changed to /home/marty/foobar and will not be changed
again unless you init another NERD tree with a similar command.

If the option is set to 2 then it behaves the same as if set to 1 except that
the CWD is changed whenever the tree root is changed. For example, if the CWD
is /home/marty/foobar and you make the node for /home/marty/foobar/baz the new
root then the CWD will become /home/marty/foobar/baz.
```

TL;DR:
```
echo 'let NERDTreeChDirMode = 2' >> ~/.vimrc
# then, within vim/NERDTree, select a git managed dir/file, and press
CR
```

**Comment by cemsbr** [comment]
Thank you. I'm closing this issue due to your solution.
